packages/lib/src/components/PaywallRenderer.svelte

```diff
@@ -22,7 +23,7 @@
   ] as IconName[];
 
   type Props = {
-    t: { [key: string]: string };
+    t: Translate;
     horizontal?: boolean;
     paywall: { [key: string]: any }; // TODO: type define paywall props
```

Comment:
This is egentligen on me :) forgot that `PaywallProps` was added.

Feel free to swap this to:
```
type Props = PaywallProps & {
  t: Translation;
  horizontal?: boolean;
  }
```
```

Reply:
It seems that this file is expecting the fetched paywall json rather than the PaywallProps?

Reply:
Oh that's correct, I forgot.

Don't mind my comment here 😅

However we _should_ get our hands on the correct Type definition as my TODO comment states.

Don't know if we want to do that programatically somehow or just copy it over from monorepo.

This has been something bothering our frontends since day 1.

Reply:
If we pull this from sesamy-js it should really have the correct type..

Reply:
Yea I guess it makes sense that sesamy-js eventually will serve the Paywall json data.

But the problem will still be there: how does sesamy-js keep the types in sync with what happens in monorepo? 😅